local-pipeline.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env sh
```

Comment:
i like the concept, though this won't work for our windows folks unless they're using WSL i think. i'll probably iterate on this later.

Reply:
That's true, I wouldn't know where to begin with a PowerShell equivalent and my .bat file experience is from the 90s 😬